nixos/tests/all-tests.nix

```diff
@@ -36,12 +36,21 @@ let
     if elem system systems then handleTest path args
     else {};
 
-  nixosLib = import ../lib {
+  nixosLibExperimental = import ../lib {
     # Experimental features need testing too, but there's no point in warning
     # about it, so we enable the feature flag.
     featureFlags.minimalModules = {};
   };
-  evalMinimalConfig = module: nixosLib.evalModules { modules = [ module ]; };
+  nixosLib = import ../lib {};
+
+  inherit
+    (nixosLibExperimental)
+    evalModules;
```

Comment:
An `evalModules` variable without a parent attrset to disambiguate it is not so great. It's not `lib.evalModules` after all, and that we be all too easy for the reader to assume.

Reply:
Yep, this is an easy fix!